charts/mission-control/templates/mission-control.yaml

```
@@ -556,67 +558,62 @@ spec:
   {{- if .Values.isAgent }}
   - name: Upstream Sync
     icon: helm
-    properties:
-      - name: Check Status Backlog
-        lookup:
-          postgres:
-            - query: SELECT reltuples FROM  pg_class  WHERE relname ='check_statuses_is_pushed_idx'
-              connection: "connection://{{ .Values.connectionName }}"
-              display:
-                expr: results.rows[0].reltuples
-      - name: Config Analysis Backlog
-        lookup:
-          postgres:
-            - query: SELECT count(*) FROM config_analysis where is_pushed is false
-              connection: "connection://{{ .Values.connectionName }}"
-              display:
-                expr: results.rows[0].count
-      - name: Config Changes Backlog
-        lookup:
-          postgres:
-            - query: SELECT count(*) FROM config_changes where is_pushed is false
-              connection: "connection://{{ .Values.connectionName }}"
-              display:
-                expr: results.rows[0].count
+    components:
+    - name: UpstreamGroup
+      lookup:
+        postgres:
+        - query: |
+            SELECT
+               (SELECT reltuples FROM pg_class WHERE relname ='check_statuses_is_pushed_idx') as cs,
+               (SELECT count(*) FROM config_analysis where is_pushed is false) as ca,
+               (SELECT count(*) FROM config_changes where is_pushed is false) as cc
+          connection: "connection://{{ .Values.connectionName }}"
+          display:
+            expr: |
+              [
+                {'name': 'Check Status Backlog', 'icon': 'sync', 'health': results.rows[0].cs < 500 ? 'healthy' : 'unhealthy' ,'properties': [{'name': 'size', 'value': results.rows[0].cs, 'headline': true}]},
+                {'name': 'Config Analysis Backlog', 'icon': 'sync', 'health': results.rows[0].ca < 200 ? 'healthy' : 'unhealthy' ,'properties': [{'name': 'size', 'value': results.rows[0].ca, 'headline': true}]},
+                {'name': 'Config Changes Backlog', 'icon': 'sync', 'health': results.rows[0].cc < 200 ? 'healthy' : 'unhealthy' ,'properties': [{'name': 'size', 'value': results.rows[0].cc, 'headline': true}]},
+              ].toJSON()
     checks:
       - inline:
-          schedule: '@every 30s'
+          schedule: '@every 2m'
           postgres:
             - query: SELECT time_end FROM job_history WHERE name = 'ReconcileAll' AND status = 'SUCCESS' LIMIT 1
               connection: "connection://{{ .Values.connectionName }}"
-              name: Last Full Reconcile
+              name: Last Full Reconcile: {{.Values.topologyName}}
```

```suggestion
              name: Last Full Reconcile/{{.Values.topologyName}}
```

This will allow them to be grouped in the UI